### PR TITLE
upgrade insecure requests was added in Edge 17

### PR DESCRIPTION
**Pull request by yatil**
See https://caniuse.com/#feat=upgradeinsecurerequests & https://developer.microsoft.com/en-us/microsoft-edge/platform/status/cspupgradeinsecurerequestsdirective/

### CODE REVIEW

File: http/headers/upgrade-insecure-requests.json [hunk 1]
```diff
@@ -15,8 +15,7 @@
               "version_added": "44"
             },
             "edge": {
-              "version_added": false,
-              "notes": "<a href='https://developer.microsoft.com/en-us/microsoft-edge/platform/status/cspupgradeinsecurerequestsdirective'>Under consideration</a> for future release."
+              "version_added": "17",
```
Comment (Elchi3) [inline http/headers/upgrade-insecure-requests.json]: Please remove the [trailing comma](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) (not allowed in JSON :confused:).

**Comment by yatil** [comment]
@Elchi3 Done!

**Comment by jlbooker** [comment]
@Elchi3 Has this been published to the site yet?  https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/upgrade-insecure-requests is still showing "NO" with the red X for Edge.

**Comment by Elchi3** [comment]
Uhm hm, you're right, @jlbooker! This PR updated the header with the same name (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Upgrade-Insecure-Requests#Browser_compatibility) but not the CSP directive. Can you file an issue or submit a PR to update https://github.com/mdn/browser-compat-data/blob/master/http/headers/content-security-policy.json#L1692?

**Pull request merged**